trade-finance-manager-api/src/v1/validation/route-validators/route-validators.js

```diff
@@ -0,0 +1,23 @@
+const { param } = require('express-validator');
+
+const userParamEscapingSanitization = param('user').isString('User ID must be a string').escape();
+const userParamMongoIdcValidation = param('user').isMongoId().withMessage('The User ID (user) provided should be a Mongo ID');
+const facilityIdMongoIdValidation = param('facilityId').isMongoId().withMessage('The Facility ID (facilityId) provided should be a Mongo ID');
```

Comment:
we won't need the "mongoId" part. we can keep it as: `facilityIdValidation` - same for the other vars